src/main/java/sirius/web/security/oauth/ReceivedTokens.java

```diff
@@ -22,10 +22,12 @@
  * @param accessToken           the access token received from the authorization server
  * @param refreshToken          the refresh token received from the authorization server
  * @param type                  the type of the tokens received from the authorization server, e.g. "Bearer" or "MAC"
+ * @param accessTokenExpiresAt  the date at which the access token expires, might be null if the server response
+ *                              contains no information
```

Comment:
Are you sure this field is `null` if the server response does not contain such information? At a quick glance, your code below seems to fall back to _expires in_ = 0, leading to _expires at_ to equal "now". Or am I wrong?

Reply:
You are right. Thanks for the finding!